doc/source/structures/misc/time.rst

```diff
@@ -32,6 +32,8 @@ Special variable TIME
 
     Any time you perform arithmetic on **TIME** you get a result back that is also a :struct:`TimeSpan`. In other words, TIME is a :struct:`TimeSpan`, but TIME + 100 is also a :struct:`TimeSpan`.
 
+.. highlight:: none
+
     Note that Kerbals do not have the concept of "months"::
```

Comment:
Correct.  It's turning kerboscirpt highlighting off, doing the block, then turning it back on.  This block is not quite kerboscipt syntax so the colored highlighting was borken.

I wish it was better "nested" than that ("please return highlighting to whatever it was" rather than picking the highlight explicitly and leaving it globally that way until picking a different highlight explicitly.)  But that seems to be the way it works.

As to it throwing an error - it didn't when I ran it.  I'll be on the lookout for it later when running it again, but it accepted it just fine when I ran it, and did produce the output file.